portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -103,7 +102,20 @@ function populateDropdown(list, ID) {
   });
 }
 
+// inserts a functioning searchbar into the navigation bar of a page.
 function insertSearch() {
+  const searchVisual = createSearch();
```

Comment:
nit: Maybe call this searchElement and createSearchElement?

Reply:
I transferred this comment to relevant repository.